int64 nr_rollouts
float64 lambda
float64 h
float64 step_size
float64 caching_factor
float64 horizon

bool adaptive_sampling
mppi_ros/Array input_variance

bool filtering
float64 cost_ratio
float64 discount_factor

bool verbose
bool debug_print

bool bound_input
mppi_ros/Array input_min
mppi_ros/Array input_max

int64 filter_window
int64 filter_order
